src/core_plugins/markdown_vis/public/markdown_vis.less

```diff
@@ -5,26 +5,13 @@
   width: 100%;
 }
 
-.vis-editor {
-
-  &.vis-type-markdown {
-
-    .visualization-options {
-      .flex-parent();
-      flex: 1 1 auto;
-    }
-
-    .markdown-vis-options {
-      .flex-parent();
-      flex: 1 1 auto;
-
-      textarea {
-        flex: 1 1 auto;
-        resize: none;
-      }
-
-    }
-
-  }
+.markdown-vis-editor,
+.markdown-vis-editor-row {
+  display: flex;
+  flex: 1 1 100%;
+}
 
+.markdown-vis-editor textarea {
```

Comment:
We could instead of needing the nested query, just using `className` to apply a class name to the `EuiTextArea` instead.